package.json

```diff
@@ -141,6 +146,37 @@
           "marc-records-editor.links.suggestion.post"
         ],
         "visible": true
+      },
+      {
+        "permissionName": "ui-quick-marc.settings.lccn-duplicate-check.retrieving",
+        "displayName": "Getting the LCCN duplicate flag",
+        "subPermissions": [
+          "mod-settings.entries.collection.get",
+          "mod-settings.entries.item.get",
+          "mod-settings.global.read.ui-quick-marc.lccn-duplicate-check"
+        ],
+        "visible": false
+      },
+      {
+        "permissionName": "ui-quick-marc.settings.lccn-duplicate-check.edit",
+        "displayName": "Edit, View: Enable duplicate LCCN (010 $a) checking of MARC bibliographic and authority records",
+        "subPermissions": [
+          "ui-quick-marc.settings.lccn-duplicate-check.retrieving",
+          "mod-settings.entries.item.post",
+          "mod-settings.entries.item.put",
+          "mod-settings.global.write.ui-quick-marc.lccn-duplicate-check"
+        ],
+        "visible": true
+      },
+      {
+        "permissionName": "mod-settings.global.read.ui-quick-marc.lccn-duplicate-check",
```

Comment:
Are `mod-settings.global.read` and `mod-settings.global.write` prefixes required by mod-settings?

Reply:
Also, what is the purpose of these two permissions?

Reply:
> Are mod-settings.global.read and mod-settings.global.write prefixes required by mod-settings?

Yes, they are required to have permission to read and write to global settings, creating a new setting in `mod-settings` requires these prefixes and `scope` according to [doc](https://github.com/folio-org/mod-settings/blob/a7239e59328fb515dc6fa65053a105e2c10bc5b5/README.md?plain=1#L35). 

> Also, what is the purpose of these two permissions?

They allow reading and writing this particular setting.

Reply:
The `settings/entrie` endpoint requires a `key` and `scope`, the `key` can be anything, the `scope` have to match the scope in `mod-settings.global.read.{scope}`.

Reply:
Ok, I understand now. I think the displayName is confusing - if the permission is created just for one scope then let's add it to displayName. Maybe `Settings (MARC): read LCCN duplicate check settings` and `Settings (MARC): write LCCN duplicate check settings`

Reply:
I changed the `displayName`, also added the related PR https://github.com/folio-org/ui-marc-authorities/pull/405